uisdk/src/main/java/tech/dojo/pay/uisdk/presentation/components/LocaleManager.kt

```diff
@@ -0,0 +1,33 @@
+package tech.dojo.pay.uisdk.presentation.components
+
+import android.annotation.TargetApi
+import android.content.Context
+import android.content.res.Configuration
+import android.os.Build
+import java.util.Locale
+
+object LocaleManager {
+
+    fun updateBaseContextLocale(context: Context, language: String?): Context {
+        if (language.isNullOrEmpty()) return context
+        val locale = Locale(language)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+            return updateResourcesLocale(context, locale)
+        }
+        return updateResourcesLocaleLegacy(context, locale)
+    }
+
+    @TargetApi(Build.VERSION_CODES.N)
```

Comment:
this will crash for older version can we mention that on the docs or to see if there are any other way that compatible with older version or maybe update our sdk

Reply:
I've tested the usage of the method above `updateBaseContextLocale` on android M and works correctly. 
This method below is private and used only inside `updateBaseContextLocale`.
@TargetApi should an annotation that tell this method will be running only on andorid N or above.
What do you think?